rust-runtime/aws-smithy-http-server/examples/pokemon-service/Cargo.toml

```diff
@@ -51,7 +51,6 @@ pokemon-service-server-sdk = { path = "../pokemon-service-server-sdk/" }
 assert_cmd = "2.0"
 home = "0.5"
 serial_test = "0.7.0"
-wrk-api-bench = "0.0.8"
```

Comment:
You also have to remove the benchmark file from the tests folder.

Reply:
Done in https://github.com/awslabs/smithy-rs/pull/2331/commits/28cb47146d2b7c67ee69f12e1cd75f3b2e54ab85.